vars/buildChain.groovy

```diff
@@ -0,0 +1,25 @@
+import groovy.json.JsonSlurper
+import org.yaml.snakeyaml.Yaml
+
+/*
+* It gets buildChain verion from composite action.yml file
+*/
+def getBuildChainVersionFromCompositeActionFile(String actionFilePath = '.ci/actions/build-chain/action.yml', String usesContainingString = 'github-action-build-chain@') {
+    def actionFileContent = readFile actionFilePath
+    def actionObject = new Yaml().load(actionFileContent)
+
+    def uses = actionObject.runs.steps.uses
+    def action = uses != null ? uses.find({ it.contains(usesContainingString) }) : null
+    if (action == null) {
+        throw new RuntimeException("There's not steps with 'uses' for build-chain ${usesContainingString}")
+    }
+
+    def buildChainScmRevision = action.substring(action.indexOf('@') + 1)
+    try {
+        def url = new URL("https://raw.githubusercontent.com/kiegroup/github-action-build-chain/${buildChainScmRevision}/package.json")
+        def packageJson = url.getText()
+        return new JsonSlurper().parseText(packageJson).version
```

Comment:
Can you explain this last part of the method?
Isn't the version already the `buildChainScmRevision` value? Don't the tag/branch corresponds with the version of action?
I was expecting that parsing the URL was enough without the need to download and parse  `package.json` 🤔